baseframe/Makefile

```diff
@@ -3,7 +3,8 @@ all: tinymce codemirror
 
 tinymce3_zip = tinymce_3.5.11_jquery.zip
 tinymce4_zip = tinymce_4.1.7_jquery.zip
-codemirror_zip = codemirror.zip
+codemirror4_zip = 4.11.0.zip
+codemirror5_zip = 5.8.0.zip
```

Comment:
We don't need both 4 and 5 (unlike TinyMCE 3 and 4, which represent an incompatible upgrade).